data/lazy_list/basic.lean

```diff
@@ -0,0 +1,51 @@
+/-
+Copyright (c) 2018 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon
+
+Traversable instance for lazy_lists.
+-/
+
+import category.traversable.equiv category.traversable.instances data.lazy_list
```

Comment:
`data.lazy_list` lives in the Lean repository. I guess we should use the same naming convention as for `vector2` and call it `data.lazy_list2`. Until this stuff is gone in Lean4...

Reply:
there is no `data.set` or `data.list` in the Lean repository, the files are in `init.data.set` and `init.data.list`. That's why only `vector` is `vector2`.

Reply:
I had to check for myself. I could have sworn that they existed. Did they use to exist?

In any case, you're right that there aren't two naming convention. I'll comply. I must say though, I'm not fond of the `vector2` naming convention. I'd prefer something like `data.vector.mathlib` for instance.